etc/DependencyInstaller.sh

```diff
@@ -86,20 +86,33 @@ _installUbuntuPackages() {
     if _versionCompare $1 -ge 23.04; then
         apt-get install klayout python3-pandas
     else
-        if [[ $1 == 20.04 ]]; then
-            klayoutChecksum=15a26f74cf396d8a10b7985ed70ab135
-        else
-            klayoutChecksum=db751264399706a23d20455bb7624264
-        fi
+        arch=$(uname -m)
         lastDir="$(pwd)"
         # temp dir to download and compile
         baseDir=/tmp/installers
         mkdir -p "${baseDir}"
-        cd ${baseDir}
-        wget https://www.klayout.org/downloads/Ubuntu-${1%.*}/klayout_${klayoutVersion}-1_amd64.deb
-        md5sum -c <(echo "${klayoutChecksum} klayout_${klayoutVersion}-1_amd64.deb") || exit 1
-        dpkg -i klayout_${klayoutVersion}-1_amd64.deb
-        cd ${lastDir}
+        cd "${baseDir}"
+        if [[ $arch == "aarch64" ]]; then
+        if [ ! -f /usr/local/bin/klayout ]; then
+            echo "Installing KLayout for aarch64 architecture"
+            installDir="/usr/local/bin"
+            git clone https://github.com/KLayout/klayout.git
+            cd klayout
+            ./build.sh -bin "${installDir}"
```

Comment:
Doesn't KLayout have several additional dependencies that need to be installed in order to get it to build?

Reply:
Hi @rovinski 
Thanks for your comment. Indeed, KLayout has dependencies that require installation, the necessary changes have already been pushed․ Regarding the installDir="/usr/local/bin" path defined for installation of Klayout, I included the PREFIX in a manner consistent with the approach used in the OpenROAD/etc/DependencyInstallers.sh file during the installation of various dependencies, however Klayout installation is done when the script is first time invoked from setup.sh with "--base" option which according to the help message is not compatible with "--prefix" option, so the installation will take place at "/usr/local" directory. Please let me know if there are any open questions that I can address.

Regards
Lia